tests/bson/bson-utcdatetime_error-001.phpt

```diff
@@ -1,22 +1,19 @@
 --TEST--
-BSON BSON\UTCDateTime #001 error
---INI--
-date.timezone=America/Los_Angeles
+BSON BSON\UTCDateTime requires object argument to implement DateTimeInterface
 --SKIPIF--
-<?php if (defined("HHVM_VERSION_ID")) exit("skip HHVM handles parameter parsing differently"); ?>
```

Comment:
We might have to put this one back in I suppose, if we find that it's still different with HHVM.